<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="LAPTOP-H8L9GDQJ" failures="1" tests="7" name="Account Creation" time="159.061" errors="0" timestamp="2025-07-13T01:41:45 IST">
  <testcase classname="testCases.CreateAccountTest" name="testNegativeAccountCreation" time="9.352"/>
  <testcase classname="testCases.CreateAccountTest" name="testNegativeAccountCreation" time="20.738"/>
  <testcase classname="testCases.CreateAccountTest" name="testNegativeAccountCreation" time="20.441"/>
  <testcase classname="testCases.CreateAccountTest" name="testNegativeAccountCreation" time="28.807">
    <failure type="java.lang.AssertionError" message="Expected error not found: Please enter the same value again. expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Expected error not found: Please enter the same value again. expected [true] but found [false]
at assertions.AssertUtils.assertTrue(AssertUtils.java:14)
at testCases.CreateAccountTest.testNegativeAccountCreation(CreateAccountTest.java:56)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 34 stack frames]]>
    </failure>
  </testcase> <!-- testNegativeAccountCreation -->
  <testcase classname="testCases.CreateAccountTest" name="testNegativeAccountCreation" time="24.439"/>
  <testcase classname="testCases.CreateAccountTest" name="testNegativeAccountCreation" time="18.118"/>
  <testcase classname="testCases.CreateAccountTest" name="testNegativeAccountCreation" time="8.476"/>
  <testcase classname="testCases.CreateAccountTest" name="verifyAccountCreation" time="0.0">
    <ignored/>
  </testcase> <!-- verifyAccountCreation -->
</testsuite> <!-- Account Creation -->
